src/main.zig

```diff
@@ -445,6 +484,12 @@ pub fn convertJsonToZon(jsonPath: []const u8) void { // TODO: Remove after #480
 				zonString.append(c);
 			},
 		}
+		controller_mappings_window_shown = true;
+	} else if(settings.playerName.len == 0) {
+		gui.openWindow("change_name");
+	} else {
+		startup_finished = true;
+		gui.openWindow("main");
```

Comment:
This appears to be in the wrong function.

Reply:
Probably didn't notice this when fixing merge conflicts while rebasing